src/Library/demos/Application window/main.blp

```diff
@@ -1,47 +1,39 @@
 using Gtk 4.0;
 using Adw 1;
 
-Adw.ApplicationWindow window {
+Gtk.ApplicationWindow window {
```

Comment:
no reason to use Adw.Window / Adw.ApplicationWindow if we don't want a freeform window